test/tests/api/v2_0/os_install_tests.py

```diff
@@ -195,20 +266,45 @@ def install_suse(self, version, nodes=[], options=None, payloadFile=None):
         # run the workflow
         self.__post_workflow(graph_name, nodes, body)
 
-    def install_ubuntu(self, version, payloadFile, nodes = []):
+    def install_ubuntu(self, version, payloadFile, nodes=[]):
         graph_name = 'Graph.InstallUbuntu'
         os_repo = defaults.get('RACKHD_UBUNTU_REPO_PATH', \
             self.__base + '/repo/ubuntu')
         # load the payload from the specified file
         body = {}
         body = self.__get_os_install_payload(payloadFile)
+        kargs = {}
+        # Ubuntu installer requirs from the dhcp both options:
+        # - routers
+        # - domain-name-servers
+        # using static ip instead
+        try:
+            # if node index is set then we can use hard coded static IP
```

Comment:
Ubuntu requires router option and dns server in the dhcp seerver . Adding these options breaks the installation of ESXI5.5.
the solution here is to remove the options from dhcp server and use static IP for Ubuntu